qml/pages/SettingsPage.qml

```diff
@@ -147,13 +149,25 @@ Dialog {
 
             SectionHeader{ text: qsTr("Notifications") }
 
-            IconTextSwitch {
-                id: notifications
-                text: qsTr("Enable notifications")
-                icon.source: "qrc:///images/icon-notifications.png"
-                icon.scale: Theme.iconSizeMedium/icon.width // Scale icons according to the screen sizes
-                checked: settings.enableNotifications
-                description: qsTr("%1 will send you notifications when you have a new message, a new notification or a friend request.").arg("Sailbook")
+            TextSwitch {
+                id: notifyRequests
+                text: qsTr("Notify friend requests")
+                checked: settings.notifyRequests
+                description: qsTr("%1 will send you notifications when getting a friend request.").arg("Sailbook")
+            }
+
+            TextSwitch {
+                id: notifyMessages
+                text: qsTr("Notify incoming messages")
+                checked: settings.notifyMessages
+                description: qsTr("%1 will send you notifications when receiving a new message.").arg("Sailbook")
+            }
+
+            TextSwitch {
+                id: notifyNotifications
+                text: qsTr("Notify facebook notifications")
```

Comment:
This string isn't conform with the rest of Sailbook. I suggest you use "Notify new notifications" instead.